tox.ini

```diff
@@ -6,7 +6,6 @@ usedevelop = True
 whitelist_externals=rm
 setenv =
     PYTHONPATH = {toxinidir}
-    KOLIBRI_HOME = {toxinidir}/kolibrihome_test
```

Comment:
I'm not really decided on this.. I added `os.environ["KOLIBRI_HOME"] = tempfile.mkdtemp()` in `settings.test` but this does not affect all test environments, for instance when running `kolibri start` in `tox.ini`.